HOMEWORK_8/EX5/RecursiveFunctions.cpp

```diff
@@ -0,0 +1,22 @@
+#include <iostream>
+#include "RecursiveFunctions.h"
+
+void printFromBiggerToSmallerByRecursion(int count)
+{
+    if(count < 1)
+        return;
+
+    std::cout<<count<<std::endl;
+    printFromBiggerToSmallerByRecursion(count - 1);
+}
+
+int number = 1;
+void printFromSmallerToBiggerByRecursion(int count)
+{
+    if(number > count)
+        return;
+
+    std::cout<<number<<std::endl;
+    number++;
+    printFromSmallerToBiggerByRecursion(count);
+}
```

Comment:
can be implemented without additional `number` variable

Reply:
I hope I understood correctly